modules/ds_bean_fields/ds_bean_fields.module

```diff
@@ -133,6 +139,7 @@ function _ds_bean_fields_render_html($field, $html) {
  * Hyphenate a text string.
  */
 function ds_bean_fields_hyphenate($text, $langcode = NULL) {
+  $patterns = NULL;
```

Comment:
I'm not actually sure what the variable or the function does. Codeclimate throws an issue without that there. currently line 173 uses $patterns. If I comment out that line:
`173: Avoid unused local variables such as '$patterns'. [phpmd]`